### ISSUE TITLE
[Umidigi One Max] Wifi Hotspot not working

### ISSUE THREAD

**Issue by defvs** [phhusson/treble_experimentations#589]
Hotspot not working on Umidigi One Max (MT6763T)
Can't tell a whole lot more. Just ask I will provide anything.

**Comment by nonnymoose** [comment]
Same here on the A1 Pro.

**Comment by defvs** [comment]
Just noticed that NFC is completely missing from the settings.

**Comment by thepowerfulpotato** [comment]
Same chip in doogee S80 here and same issue, WiFi hotspot won't turn on/crashes the radio. Not sure how to capture error log file though.

**Comment by ke7zds** [comment]
Same on UMi Z2.  I'm guessing this is a MTK Helio driver issue.
 Would be nice to see a fix.  Thank you!

**Comment by defvs** [comment]
Renaming this to MT6763T issues

**Comment by penn5** [comment]
It happens on MT6771 too. Renaming to MT67XX(T)

**Comment by penn5** [comment]
Wait this is a duplicate, and it has 2 issues in it. So it's invalid

**Comment by defvs** [comment]
So you're telling me than it's simpler to have one issue per phone, per non-working feature ? No wonder the issues list is crowded.

**Comment by penn5** [comment]
Yes that's exactly what I'm telling you. Look at the countless issues closed because they need to be split up.

**Comment by Jessi810** [comment]
Hotspot and USB Tethering not working (Automatically turns off) when mobile data is ON. But when mobile data is OFF, hotspot and USB tethering is working (doesn't turn off automatically). Probably a bug relating on MTK devices.

Havos-OS and MTK6761
Logcat showing errors only: [logcat_error-only.txt](https://github.com/phhusson/treble_experimentations/files/3386401/logcat_error-only.txt)
[logcat-no-filter.txt](https://github.com/phhusson/treble_experimentations/files/3386400/logcat-no-filter.txt)

**Comment by defvs** [comment]
Same for me, even with Bluetooth

**Comment by defvs** [comment]
Seems to be IPTables related. I get some iptables errors on logcat.

**Comment by chicato83** [comment]
I have the same problems and not call outgoing slot 2. Umi z2 pro

**Comment by scottvpb** [comment]
I have the same problem with Lenovo K5 Pro L38041

**Comment by defvs** [comment]
Good news !
NFC can be fixed with a magisk module. At least it worked for me : https://github.com/Magisk-Modules-Repo/NFC4PRA-GSI
No side effects for me in a week.

**Comment by defvs** [comment]
With little fiddling with the .zip magisk module, we can find what files makes the NFC work on our chips and then probably make an easy fix.

**Comment by penn5** [comment]
Wtf! That module is for Huawei...

Try adding the feature xmls separately

**Comment by defvs** [comment]
that's the weird thing. Probably like a small part of the module is making NFC work on our chips while the other part is completely useless.

**Comment by Zuy9o** [comment]
Wi-Fi doesn't work ! system-arm64-aonly-vanilla-nosu.img.xz  ,  Put EvolutionX-4.20.2 , SafetyPatch_BL9000 ,you can somehow solve the problem !?
